packages/theme/src/motion/AnimationStyles.ts

```diff
@@ -155,15 +155,15 @@ function _createAnimation(
 
 function _createSlideInX(fromX: number): string {
   return keyframes({
-    from: { transform: `translate3d(${fromX}px,0,0)` },
-    to: { transform: `translate3d(0,0,0)` },
+    from: { transform: `translate3d(${fromX}px,0,0)`, pointerEvents: 'none' },
+    to: { transform: `translate3d(0,0,0)`, pointerEvents: 'auto' },
```

Comment:
I think setting it back to `auto` at the end isn't really necessary, since it should get reverted to its previous value once the animation ends. But I'm new to css animations and wasn't sure that was guaranteed, so I added this just in case. I can remove this if it is in fact guaranteed.

Reply:
This has regressed some features within M365 where pointer events none was actually required. This took priority. We should revert this change.